### PR TITLE
Nttw4 sept updates

### PR DESCRIPTION
**Pull request by dericed**
ping @ablwr @JeromeMartinez

### CODE REVIEW

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw4.html.twig [hunk 1]
```diff
@@ -88,7 +76,7 @@ Closing: 17:00</p>
 
 <h2>Travel Grants</h2>
 
-<p>As done at the last No Time to Wait conferences, we are working with potential sponsors and voluntary contributions in order to offer travel grants to potential attendees that may need support. More info on this soon.</p>
+<p>As done at past No Time to Wait conferences, we are offering travel grants to potential attendees that may need support. Thanks in particular to Reto Kromer and AV Preservation, https://avpres.net/, for sponsoring these travel grants. Applications for travel grants shall be accepted until September 25th with announcements to recipients by September 30th. Please apply using <a href="http://bit.ly/nttw4travelgrant">our travel grant application form</a>.</p>
```
Comment (ablwr) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw4.html.twig]: I think that link should be an href under AV Preservation instead of just a url

Comment (retokromer) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw4.html.twig]: The URL should be `https://reto.ch/` not `https://avpres.net/` and the please set full name `AV Preservation by reto.ch` (for fiscal reasons).

**Comment by dericed** [comment]
the commits are updated based on comments, please re-review.

**Review by ablwr — APPROVED** [review]

**Comment by dericed** [comment]
@JeromeMartinez, anything blocking, can we prep to hype tomorrow?

**Pull request merged**